src/containers/Login/index.tsx

```diff
@@ -129,16 +133,19 @@ const Login = ({ toggleSnackbar, setSnackbarMessage, setLoading }: ILoginProps)
     )
   }
 
+  const isSmallScreen = useMediaQuery('(max-width:819px)')
```

Comment:
Do not hardcode this. Use this plz https://mui.com/material-ui/react-use-media-query/#using-material-uis-breakpoint-helpers

Reply:
Ok